src/lib/html.ts

```diff
@@ -122,6 +122,14 @@ const create_one_result_table = (data: ImplementationTable, implementers: Implem
                     },
                 },
             },
+            {
+                td : {
+                    $ : {
+                        class : row.required ? Constants.CLASS_MUST : Constants.CLASS_SHOULD,
```

Comment:
Can you put this class on the tr instead? That way the CSS can color the row differently for non-required stuff, which'll make it easier to skim.

For example:
tr.should td.pass, tr.may td.pass { background-color: lightgreen; }
tr.should td.fail, tr.may td.fail { background-color: pink; }

Reply:
Yeah... I would love if I could. The xmljs package is fairly inflexible, alas!, and I did not find a good way to do it yet. At the moment, I set the "Req" column entries to the `must`/`should`/`may` classes; maybe it will be enough.

Reply:
What I did, after all, is to

- set the class attribute on the first column entry instead of the Req column
- use the `td.should ~ td, td.should` CSS selector, which refers to the full line (as proposed on [Slack](https://w3ccommunity.slack.com/archives/G01P2RBAD7T/p1637788081050700)

At the moment, the effect is the "gray" out the corresponding rows, but we can see this later.